pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -115,6 +115,7 @@ type ResultsAPIProperties struct {
 	LoggingPluginForwarderDelayDuration *uint  `json:"logging_plugin_forwarder_delay_duration,omitempty"`
 	LoggingPluginQueryLimit             *uint  `json:"logging_plugin_query_limit,omitempty"`
 	LoggingPluginQueryParams            string `json:"logging_plugin_query_params,omitempty"`
+	LoggingPluginMultipartRegex         string `json:"logging_plugin_multipart_regex,omitempty"`
```

Comment:
Do we need to add any validation test for this ?

Reply:
I mean do we need to add any validation for this field, if so we can add it here https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonresult_validation.go

Reply:
IMO no. This is not something which applies to the TektonResult CRD, but rather configuration option stored in a ConfigMap. It matches against 3rd party service (e.g Vector) which is not managed by the operator, so it's not something we can match end to end.